.github/workflows/copy-container-values-workflow.yaml

```
@@ -31,10 +31,16 @@ jobs:
             echo "${{ github.event.inputs.env }} is not supported as an environment"
           fi
 
+      - name: Run Helm Template
+        run: |
+          echo "Running helm template with value files from values/${{ github.event.inputs.env }}/"
+          OUTPUT_FOLDER="k8s/output/${{ github.event.inputs.env }}"
+          helm template --values "values/${{ github.event.inputs.env }}/values.yaml" --values "values/${{ github.event.inputs.env }}/values-containers.yaml" --output-dir "$OUTPUT_FOLDER" k8s/
```

```suggestion
          helm template \
            --values "values/${{ github.event.inputs.env }}/values.yaml" \
            --values "values/${{ github.event.inputs.env }}/values-containers.yaml" \
            --output-dir "k8s/output/${{ github.event.inputs.env }}" \
            k8s/
```

Kunne man gjort det slik? Lettere å lese?